src/batch-transaction.ts

```diff
@@ -18,8 +18,15 @@ import {PreciseDate} from '@google-cloud/precise-date';
 import {promisifyAll} from '@google-cloud/promisify';
 import * as extend from 'extend';
 import * as is from 'is';
-import {codec} from './codec';
 import {Snapshot} from './transaction';
+import {google} from '../proto/spanner';
+import {Session} from '.';
+
+export interface TransactionIdentifier {
+  session: string | Session;
+  transaction?: string;
+  readTimestamp?: string | google.protobuf.ITimestamp;
```

Comment:
This does not match what is returned here https://github.com/googleapis/nodejs-spanner/blob/master/src/batch-transaction.ts#L322